<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>menu_dialog</class>
 <widget class="QDialog" name="menu_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>329</width>
    <height>263</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>menu_dialog</string>
  </property>
  <widget class="QLineEdit" name="EntradaA">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="SalidaB">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Conversor de bases (2 - 30)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>EntradaA</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>SalidaB</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxBaseA">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>BaseA </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>BaseB</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxBaseB">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>221</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
